GMP : checking for build system executable suffix... configure: error: Cannot determine executable suffix

Hello all,
I am new to cross compilation.I am trying to do canadian compilation for the following
build : windows10 WSL (windows subsystem for linux)
host: windows 10
target: x86_64 linux
I tried the following steps for compilation.
ct-ng x86_64-w64-mingw32
ct-ng build
PATH=~/x-tools/x86_64-w64-mingw32/bin:$PATH
ct-ng x86_64-w64-mingw32,x86_64-pc-linux-gnu
ct-ng build

while doing the step ct-ng x86_64-w64-mingw32,x86_64-pc-linux-gnu i encounterd the following issue

`[ERROR]    checking for build system executable suffix... configure: error: Cannot determine executable suffix
[ERROR]  
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Installing GMP for host'
[ERROR]  >>        called in step '(top-level)'
[ERROR]  >>
[ERROR]  >>  Error happened in: CT_DoExecLog[scripts/functions@376]
[ERROR]  >>        called from: do_gmp_backend[scripts/build/companion_libs/100-gmp.sh@92]
[ERROR]  >>        called from: do_gmp_for_host[scripts/build/companion_libs/100-gmp.sh@58]
[ERROR]  >>        called from: do_companion_libs_for_host[scripts/build/companion_libs.sh@36]
[ERROR]  >>        called from: main[scripts/crosstool-NG.sh@695]
[ERROR]  >>
[ERROR]  >>  For more info on this error, look at the file: 'build.log'
[ERROR]  >>  There is a list of known issues, some with workarounds, in:
[ERROR]  >>      https://crosstool-ng.github.io/docs/known-issues/
[ERROR]  >>
[ERROR]  >> NOTE: Your configuration includes features marked EXPERIMENTAL.
[ERROR]  >> Before submitting a bug report, try to reproduce it without enabling
[ERROR]  >> any experimental features. Otherwise, you'll need to debug it
[ERROR]  >> and present an explanation why it is a bug in crosstool-NG - or
[ERROR]  >> preferably, a fix.
[ERROR]  >>
[ERROR]  >>  If you feel this is a bug in crosstool-NG, report it at:
[ERROR]  >>      https://github.com/crosstool-ng/crosstool-ng/issues/
[ERROR]  >>
[ERROR]  >>  Make sure your report includes all the information pertinent to this issue.
[ERROR]  >>  Read the bug reporting guidelines here:
[ERROR]  >>      http://crosstool-ng.github.io/support/
[ERROR]  
[ERROR]  (elapsed: 43:36.06)
`
Please suggest how to fix the issue. Am i missing any their any configuration?

[build.zip](https://github.com/crosstool-ng/crosstool-ng/files/4433917/build.zip)

[config.txt](https://github.com/crosstool-ng/crosstool-ng/files/4433925/config.txt)

--- a/scripts/build/companion_libs/100-gmp.sh
+++ b/scripts/build/companion_libs/100-gmp.sh
@@ -118,9 +118,12 @@ do_gmp_backend() {
             ;;
     esac
 
-    # FIXME: GMP's configure script doesn't respect the host parameter
-    # when not cross-compiling, ie when build == host.
+    # GMP's configure script doesn't respect the host parameter
+    # when not cross-compiling, ie when build == host so set
+    # CC_FOR_BUILD and CPP_FOR_BUILD.
     CT_DoExecLog CFG                                \
+    CC_FOR_BUILD="${CT_BUILD}-gcc"                  \
+    CPP_FOR_BUILD="{CT_BUILD}-cpp"                  \
     CC="${host}-gcc"                                \
     CFLAGS="${cflags} -fexceptions"                 \
     LDFLAGS="${ldflags}"                            \

